source/server/spongineer/commands.rst

```diff
@@ -81,52 +84,59 @@ Command               Description                               Permission
+/ban-ip               Ban a player's IP address.                minecraft.command.ban-ip
+/banlist              View all banned players.                  minecraft.command.banlist
+/clear                Clear an inventory.                       minecraft.command.clear
+/deop                 Remove OP from a player.                  minecraft.command.deop
+/difficulty           Set the game difficulty.                  minecraft.command.difficulty
+/gamemode             Set the gamemode of a player.             minecraft.command.gamemode
+/gamerule             Set a gamerule.                           minecraft.command.gamerule
+/give                 Give an item to a player.                 minecraft.command.give
+/kill                 Kill a player or entity.                  minecraft.command.kill
+/op                   Give Operator status to a player.         minecraft.command.op
+/pardon               Remove a player from the ban list.        minecraft.command.pardon
+/save-all             Save the server.                          minecraft.command.save-all
+/save-off             Disable automatic server saving.          minecraft.command.save-off
+/save-on              Enable automatic server saving.           minecraft.command.save-on
+/setidletimeout       Define how long players can be idle       minecraft.command.setidletimeout
                       before getting kicked.
-/setworldspawn        Set the spawnpoint for the world.
-/stop                 Stop the server.
-/toggledownfall       Toggle between sunny and rainy weather.
-/tp                   Teleport players and entities.
-/weather              Set the weather to a defined condition.
-/whitelist            Manage the server whitelist.
-/worldborder          Manage the world border.
-====================  ========================================  ====================
+/setworldspawn        Set the spawnpoint for the world.         minecraft.command.setworldspawn
+/stop                 Stop the server.                          minecraft.command.stop  
+/toggledownfall       Toggle between sunny and rainy weather.   minecraft.command.toggledownfall
+/tp                   Teleport players and entities.            minecraft.command.tp
+/weather              Set the weather to a defined condition.   minecraft.command.weather
+/whitelist            Manage the server whitelist.              minecraft.command.whitelist
+/worldborder          Manage the world border.                  minecraft.command.worldborder 
+====================  ========================================  ================================
+
+In addition, there are two permissions created by Sponge for controlling the ability to edit commandblocks:
+
+* Allow editing an ordinary commandblock of the given name: minecraft.commandblock.edit.block.<name>
+* Allow editing a minecart commandblock of the given name: minecraft.commandblock.edit.minecart.<name>
```

Comment:
might want to clarify here that <name> is the name of the command block -- by default @